Model/Ui/CapabilitiesConfigProvider.php

```diff
@@ -0,0 +1,58 @@
+<?php
+namespace Omise\Payment\Model\Ui;
+
+use Magento\Checkout\Model\ConfigProviderInterface;
+use Magento\Payment\Api\PaymentMethodListInterface;
+use Magento\Store\Model\StoreManagerInterface;
+use Omise\Payment\Model\Capabilities;
+use Omise\Payment\Model\Config\Installment as OmiseInstallmentConfig;
+
+class CapabilitiesConfigProvider implements ConfigProviderInterface
```

Comment:
We can't hardcode these

Reply:
@jonrandy no no, not to hard code. This just to demonstrate the data flow.

Reply:
Yes... had discussion few times with @jonrandy about similar stuff... and I am religious pro-push stuff to work on frontend side.. whatever it can be done on frontend we should push out from backend.. that is also where `magento 2` goes..

Reply:
Yes, but we the 'stuff' and data that gets used on the front-end initially comes from the back end. There is no real point sending stuff to the front end that won't be used or still requires some initial trivial processing before use. It makes sense to send only what is needed in a form that is easily useable by the front end code

Reply:
I also don't think frontend should process or manipulate any data if it is not necessary.
Since the data is coming from Backend, then plugin should handle the data to make it in a ready-to-use form.
That is also to limit access to an unnecessary data that it won't going to be used at the frontend as well.

Reply:
Ok... well if it does not threat security stuff.. it should be done on frontend.. that is just one idea in modern development.. I love it..

Reply:
I mean.. user's computer should calculate as much stuff as possible if it does not threat security or break architecture of few other stuff that does not apply to our situation.

Reply:
Totally depends on what the processing is